pycrostates/utils/_checks.py

```diff
@@ -67,6 +67,7 @@ def __instancecheck__(cls, other):
     "path-like": (str, Path, os.PathLike),
     "int": (_IntLike(),),
     "callable": (_Callable(),),
+    "bool": (bool,),
```

Comment:
Why did you need this?

`_check_type(True, (bool,))` should work directly, no need for a string wrapper.

Reply:
You are right. I reverted this change and updated the corresponding calls in aahc.py